app/Policies/CoursePolicy.php

```diff
@@ -24,6 +24,11 @@ public function view(User $user, Course $course)
         return $course->users()->where('user_id', $user->id)->exists();
     }
 
+    public function create(User $user)
+    {
+        return false;
```

Comment:
Why is this set to false?

Reply:
Since the user is admin it will be set to true in `AuthServiceProvider`.